documents/Libraries/stdlib/stdlib_sx-osl_impl.mtlx

```diff
@@ -803,13 +803,13 @@
   <!-- ======================================================================== -->
 
   <!-- <blur> -->
-  <implementation name="IM_blur_float_sx_osl" nodedef="ND_blur_float" file="stdlib/osl/mx_blur_float.osl" function="mx_blur_float" language="sx-osl"/>
-  <implementation name="IM_blur_color2_sx_osl" nodedef="ND_blur_color2" file="stdlib/osl/mx_blur_color2.osl" function="mx_blur_color2" language="sx-osl"/>
-  <implementation name="IM_blur_color3_sx_osl" nodedef="ND_blur_color3" file="stdlib/osl/mx_blur_color.osl" function="mx_blur_color3" language="sx-osl"/>
-  <implementation name="IM_blur_color4_sx_osl" nodedef="ND_blur_color4" file="stdlib/osl/mx_blur_color4.osl" function="mx_blur_color4" language="sx-osl"/>
-  <implementation name="IM_blur_vector2_sx_osl" nodedef="ND_blur_vector2" file="stdlib/osl/mx_blur_vector2.osl" function="mx_blur_vector2" language="sx-osl"/>
-  <implementation name="IM_blur_vector3_sx_osl" nodedef="ND_blur_vector3" file="stdlib/osl/mx_blur_vector3.osl" function="mx_blur_vector3" language="sx-osl"/>
-  <implementation name="IM_blur_vector4_sx_osl" nodedef="ND_blur_vector4" file="stdlib/osl/mx_blur_vector4.osl" function="mx_blur_vector4" language="sx-osl"/>
+  <implementation name="IM_blur_float_sx_osl" nodedef="ND_blur_float" function="mx_blur_float" language="sx-osl"/>
+  <implementation name="IM_blur_color2_sx_osl" nodedef="ND_blur_color2" function="mx_blur_color2" language="sx-osl"/>
+  <implementation name="IM_blur_color3_sx_osl" nodedef="ND_blur_color3" function="mx_blur_color3" language="sx-osl"/>
+  <implementation name="IM_blur_color4_sx_osl" nodedef="ND_blur_color4" function="mx_blur_color4" language="sx-osl"/>
+  <implementation name="IM_blur_vector2_sx_osl" nodedef="ND_blur_vector2" function="mx_blur_vector2" language="sx-osl"/>
+  <implementation name="IM_blur_vector3_sx_osl" nodedef="ND_blur_vector3" function="mx_blur_vector3" language="sx-osl"/>
+  <implementation name="IM_blur_vector4_sx_osl" nodedef="ND_blur_vector4" function="mx_blur_vector4" language="sx-osl"/>
```

Comment:
Nitpick: for other nodes that have a C++ implementation we exclude the function attribute as well, it's not used.